src/status.cc

```diff
@@ -0,0 +1,243 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
@colincross Are you employed by google?

Reply:
Looks like all of the files have a copyright header. I think this just maintains consistency with existing implementation files. I was unable to find anything in the source indicating a requirement to sign an agreement around copyright, or how to specify copyright on new files, so I'm not sure what the team maintaining ninja expect in this regard.

Reply:
The official project rules are that the copyright line says copyright Google. This is completely wrong unless the person who wrote the file is either employed by Google, or signs a copyright transfer agreement. 

You're not going to get a different answer from the official project maintainers. So you'll have to make a judgment call 

Since @colincross submitted the original commit for this file, its not clear to me that changing it to something else would be appropriate either.

Its a crappy project rule that results in this weird outcome.

Reply:
Sorry, missed this.  Yes, I contributed this as a Google employee.